pyunlocbox/functions.py

```diff
@@ -830,7 +842,10 @@ def _prox(self, x, T):
         # Tight frame.
         if self.tight:
             tmp1 = self.A(x) - self.y()
-            scale = self.epsilon / np.sqrt(np.sum(tmp1 * tmp1, axis=0))
+            with np.errstate(divide='ignore', invalid='ignore'):
+                # Avoid 'division by zero' warning
+                scale = np.true_divide(self.epsilon,
```

Comment:
They are not to be feared all the time :) in this case true_division returns `np.inf` when you divide by zero, but the next line uses `np.min(1, scale)`, so it doesn't matter that the scale is `np.inf`. You'd like me to add this import here, or at the top of the module?